src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/AssetsComputingHelper.cs

```diff
@@ -65,6 +70,11 @@ public class AssetsComputingHelper
         };
 
         return reason != null;
+
+        bool IsDefaultIcuMode()
+        {
+            return !invariantGlobalization && !loadFullICUData && !hybridGlobalization && string.IsNullOrEmpty(customIcuCandidateFilename);
+        }
```

Comment:
can be a statement expression:
```suggestion
        bool IsDefaultIcuMode() =>
                !invariantGlobalization &&
                !loadFullICUData &&
                !hybridGlobalization &&
                string.IsNullOrEmpty(customIcuCandidateFilename);
```